package.json

```
@@ -56,6 +56,7 @@
     "test": "turbo run testing check-engines --continue",
     "test-sync": "npm run jest && npm run test-run",
     "test-run": "(cd test/ && node ./run.mjs)",
+    "test-check-rate-limit-snapshots": "(cd test/ && node ./check-rate-limit-snapshots.mjs)",
     "test-run-record": "(cd test/ && node ./run.mjs record)",
```

This should ensure that rate-limited snapshots don't show up.

```suggestion
    "test-run-record": "(cd test/ && node ./run.mjs record && node ./check-rate-limit-snapshots.mjs)",
```

If we also want to run it in CI though, perhaps this would be better?

```suggestion
    "test-check-rate-limit-snapshots": "(cd test/ && node ./check-rate-limit-snapshots.mjs)",
    "test-run-record": "(cd test/ && node ./run.mjs record) && npm run test-check-rate-limit-snapshots",
```

---

Sounds great :+1: